src/main/java/org/folio/rs/domain/entity/AccessionQueueRecord.java

```diff
@@ -34,4 +34,34 @@
   private String instanceTitle;
 
   private String instanceAuthor;
+
+  private String instanceCreator;
+
+  private String instanceContributors;
+
+  private String publisher;
+
+  private String publishYear;
+
+  private String publishPlace;
+
+  private String volume;
+
+  private String enumeration;
+
+  private String chronology;
+
+  private String issn;
```

Comment:
ISSN, ISBN, OCLC: can we introduce format validation? I mean we need to handle saved values to be sure that AccessionQueueRecord contains valid identifier values?